EasyPost/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("EasyPost")]
+[assembly: AssemblyDescription("EasyPost .NET/.NET Core Shipping API Client Library for .NET https://easypost.com/docs")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("EasyPost")]
+[assembly: AssemblyProduct("EasyPost")]
+[assembly: AssemblyCopyright("Copyright © 2022")]
```

Comment:
This should match the LICENSE probably.

Reply:
Resolved in [17ae1c2](https://github.com/EasyPost/easypost-csharp/pull/236/commits/17ae1c2e7f7dab18144c13c992d365649e6985d2)